src/db.rs

```diff
@@ -327,12 +327,19 @@ impl DbInner {
                             guard.memtable().put(
                                 kv.key.clone(),
                                 value.clone(),
-                                kv.attributes.clone(),
+                                RowAttributes {
```

Comment:
why are we removing the `ts` in this situation instead of cloning the attributes? if the idea is not to encode them in the data that should be in the codec, it's probably fine for memtables to always have the create_ts for now so we when we make it configurable we don't forget to update this and get confused about what's going on

Reply:
I didn't understand the second part. +1 on retaining the `ts` from wal.

Reply:
sorry it's a lost-update after a refactor inside this pr, this is fixed in 54b3991